File: samples/client/petstore/java/src/main/java/org/openapijsonschematools/components/headers/refschemaheader/RefSchemaHeaderSchema.java
package org.openapijsonschematools.components.headers.refschemaheader;
import org.openapijsonschematools.configurations.SchemaConfiguration;
import org.openapijsonschematools.schemas.JsonSchema;
import org.openapijsonschematools.schemas.JsonSchemas;
import org.openapijsonschematools.schemas.FrozenList;
import org.openapijsonschematools.schemas.FrozenMap;

import java.time.LocalDate;
import java.time.ZonedDateTime;
import java.util.AbstractMap;
import java.util.LinkedHashMap;
import java.util.LinkedHashSet;
import java.util.List;
import java.util.Map;
import java.util.Set;
import org.openapijsonschematools.components.schemas.StringWithValidation;

public class RefSchemaHeaderSchema {
    // $refed class
    
    
    public class RefSchemaHeaderSchema1 implements JsonSchema {
        public static String validate(String arg, SchemaConfiguration configuration) {
            return JsonSchema.validate(StringWithValidation.StringWithValidation1.class, arg, configuration);
        }
    }
}

File: samples/client/petstore/kotlin/src/main/kotlin/org/openapijsonschematools/client/components/headers/refschemaheader/RefSchemaHeaderSchema.kt
package org.openapijsonschematools.client.components.headers.refschemaheader;

import org.checkerframework.checker.nullness.qual.Nullable;
import org.openapijsonschematools.client.components.schemas.StringWithValidation;

public class RefSchemaHeaderSchema extends StringWithValidation {
    // $refed class
    
    
    public static class RefSchemaHeaderSchema1 extends StringWithValidation1 {
        private static @Nullable RefSchemaHeaderSchema1 instance = null;
        public static RefSchemaHeaderSchema1 getInstance() {
            if (instance == null) {
                instance = new RefSchemaHeaderSchema1();
            }
            return instance;
        }
    }
}
